common/artifacts/901_wc_artifacts_weapon_twohanded.txt

```diff
@@ -522,5 +458,26 @@ great_royal_sword = {
 	}
 
 	picture = "GFX_great_royal_sword"
+	slot = weapon
+}
+
+# Common
+tauren_totem = {
+	quality = 1
+	flags = { combat_weapon mace }
+
+	active = {
+		shared_physical_weapon_trigger = yes
+		culture_group = tauren_group
```

Comment:
I would make it for `is_large_humanoid_trigger` only.

Reply:
done